src/components/IKImage/combinedProps.ts

```diff
@@ -0,0 +1,15 @@
+import { UrlOptionsPath, UrlOptionsSrc } from 'imagekit-javascript/dist/src/interfaces/UrlOptions';
+import { InferProps } from 'prop-types';
+import COMMON_PROPS from "../IKContext/props";
+import Props from './props';
+
+const COMBINED_IMAGE_PROP_TYPES = {
```

Comment:
In a typescript project `<IKImage>` is showing error
```
IKImage urlEndpoint={urlEndpoint} src={uploadedImageSource} />
```
<img width="869" alt="image" src="https://user-images.githubusercontent.com/45416977/190080119-5eaeab31-2650-48a7-8eda-7dd133d0161d.png">
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/45416977/190080222-8e12a4aa-6b41-40c8-81bf-0a6e7d9d85ee.png">

Reply:
What is the value of uploadedImageSource ? From the screenshot it seems to be undefined. In which case this failure seems correct. Because, either src or path needs to be provided.

Reply:
It will be set after upload `onSuccess`. Here is how my code looks like at the moment.
```
const onSuccess = (res: any) => {
    debugger;
    console.log("Success");
    console.log(res);
    setUploadedImageSource(res.url);
  };
  const [uploadedImageSource, setUploadedImageSource] = useState();
```